eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -68,12 +68,13 @@ stages:
         name: ${{ variables.defaultPoolName }}
         demands: ${{ variables.defaultPoolDemands }}
       container: ${{ parameters.centOSStream8Container }}
-      bootstrapPrep: false          # 🚫
-      enablePoison: false           # 🚫
-      excludeSdkContentTests: true  # ✅
-      excludeOmniSharpTests: true   # ✅
-      runOnline: true               # ✅
-      buildFromArchive: false       # 🚫
+      bootstrapPrep: false               # 🚫
+      enablePoison: false                # 🚫
+      excludeSdkContentTests: true       # ✅
+      excludeOmniSharpTests: true        # ✅
+      runOnline: true                    # ✅
+      buildFromArchive: false            # 🚫
```

Comment:
Would you mind alphabetizing this list?  It was at one point and recently got out of order.